vtysh/bgp_vty.c

```diff
@@ -5182,13 +5229,21 @@ DEFUN(no_neighbor_timers,
     get_bgp_neighbor_with_bgp_router_and_ipaddr(bgp_router_context, ip_addr);
 
     if (ovs_bgp_neighbor) {
-        key_timers[0] = "Keepalive";
-        key_timers[1] = "Holdtimer";
-        tim_val.keepalive = 0;
-        tim_val.holdtime = 0;
-        /* To write to ovsdb nbr table. */
-        ovsrec_bgp_neighbor_set_timers(ovs_bgp_neighbor, key_timers,
-                                       (int64_t *)&tim_val,0);
+        if (ovs_bgp_neighbor->bgp_peer_group) {
+	    VLOG_ERR(OVSDB_TXN_COMMIT_ERROR);
+            VLOG_ERR("Error in transaction for setting timers of neighbor."
+                     "Neighbor %s already has been assigned to the peer group.\n",
+                                                                        argv[0]);
+            ERRONEOUS_DB_TXN(txn, "%% Unable to set timers. Neighbor already has been"
+	                                              " assigned to the peer group.");
+        } else {
+            tim_val.keepalive = 0;
+            tim_val.holdtime = 0;
+            /* To write to ovsdb nbr table. */
+            neighbor_set_timers(ovs_bgp_neighbor, &tim_val, 0);
```

Comment:
I get this logic from previous realisation of no_neighbor_timers. 
If we change this value 0 -> 2, we will have wrong output some commands, wrong running-config.
n_timers == 0 suggests that we use default values